tool/jt.rb

```diff
@@ -756,9 +765,31 @@ def test_mri(*args)
                       else
                         []
                       end
-
       files_to_run = (include_files - exclude_files)
+      
+    end
+
+    cext_tests = files_to_run.select { |f| f.include?("cext/ruby") }
```

Comment:
It seems better to set a variable above when args includes `--cext`, unless this should be triggered in other cases?